lib/modules/platform/bitbucket/pr-cache.ts

```diff
@@ -74,19 +74,19 @@ export class BitbucketPrCache {
     return prCache.getPrs();
   }
 
-  private addPr(pr: Pr): void {
+  private setPr(pr: Pr): void {
     logger.debug(`Adding PR #${pr.number} to the PR cache`);
```

Comment:
needs a change. this renaming should be a separate PR

Reply:
https://github.com/renovatebot/renovate/pull/33174